db/schema.rb

```diff
@@ -31,13 +31,13 @@
     t.integer "selection_type", default: 0, null: false
     t.integer "min_players_per_team"
     t.integer "max_players_per_team"
+    t.text "image_data"
     t.integer "matchtype"
     t.integer "bestof_length", default: 1
     t.integer "game_winrule"
     t.integer "points_for_win", default: 3
     t.integer "points_for_draw", default: 1
     t.integer "points_for_lose", default: 0
-    t.text "image_data"
```

Comment:
I'd rather not have this in here... But maybe it will resolve anyways when merging a newer dev.